2.1_first_module.ipynb

```diff
@@ -200,26 +192,21 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "collapsed": false
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
-    "// Scala Code: Calling Driver to instantiate Passthrough + PeekPokeTester and execute the test.\n",
+    "// Scala Code: `test` runs the unit test. \n",
     "// Don't worry about understanding this code; it is very complicated Scala.\n",
```

Comment:
Does this comment still hold now that the driver magic is significantly reduced?

Reply:
I will get rid of this comment, I do think the magic is reduced